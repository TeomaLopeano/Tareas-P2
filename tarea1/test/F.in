insertarEnColCadenas 8 8.0 2
insertarEnColCadenas 8 18.0 2
insertarEnColCadenas 8 28.0 2
insertarEnColCadenas 8 38.0 2
insertarEnColCadenas 5 5.0 2
insertarEnColCadenas 8 8.0 9
insertarEnColCadenas 8 18.0 9
insertarEnColCadenas 8 28.0 9
insertarEnColCadenas 8 38.0 9
cantidadColCadenas 2
infoEnColCadenas 8 2
removerDeColCadenas 8 2
infoEnColCadenas 8 2
infoEnColCadenas 8 9
removerDeColCadenas 8 9
removerDeColCadenas 8 9
infoEnColCadenas 8 2
infoEnColCadenas 8 9
removerDeColCadenas 8 2
removerDeColCadenas 8 2
infoEnColCadenas 8 2
infoEnColCadenas 8 9
cantidadColCadenas 2
cantidadColCadenas 5
cantidadColCadenas 9
removerDeColCadenas 8 2
infoEnColCadenas 5 2
removerDeColCadenas 5 2
cantidadColCadenas 2
Fin