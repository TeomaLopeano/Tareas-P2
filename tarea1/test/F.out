1>Se insertó (8,8.00) en la cadena 2.
2>Se insertó (8,18.00) en la cadena 2.
3>Se insertó (8,28.00) en la cadena 2.
4>Se insertó (8,38.00) en la cadena 2.
5>Se insertó (5,5.00) en la cadena 2.
6>Se insertó (8,8.00) en la cadena 9.
7>Se insertó (8,18.00) en la cadena 9.
8>Se insertó (8,28.00) en la cadena 9.
9>Se insertó (8,38.00) en la cadena 9.
10>La cantidad en la cadena 2 es 5.
11>La primera aparición de 8 en la cadena 2 es (8,38.00)
12>Se removió la primera aparición de 8 de la cadena 2.
13>La primera aparición de 8 en la cadena 2 es (8,28.00)
14>La primera aparición de 8 en la cadena 9 es (8,38.00)
15>Se removió la primera aparición de 8 de la cadena 9.
16>Se removió la primera aparición de 8 de la cadena 9.
17>La primera aparición de 8 en la cadena 2 es (8,28.00)
18>La primera aparición de 8 en la cadena 9 es (8,18.00)
19>Se removió la primera aparición de 8 de la cadena 2.
20>Se removió la primera aparición de 8 de la cadena 2.
21>La primera aparición de 8 en la cadena 2 es (8,8.00)
22>La primera aparición de 8 en la cadena 9 es (8,18.00)
23>La cantidad en la cadena 2 es 2.
24>La cantidad en la cadena 5 es 0.
25>La cantidad en la cadena 9 es 2.
26>Se removió la primera aparición de 8 de la cadena 2.
27>La primera aparición de 5 en la cadena 2 es (5,5.00)
28>Se removió la primera aparición de 5 de la cadena 2.
29>La cantidad en la cadena 2 es 0.
30>Fin.
